stdlib/3/io.pyi

```diff
@@ -102,7 +103,10 @@ class BytesIO(BinaryIO):
     # TODO should be the next line instead
     # def writelines(self, lines: List[Union[bytes, bytearray]]) -> None: ...
     def writelines(self, lines: Any) -> None: ...
-    def readline(self, size: int = ...) -> bytes: ...
+    if sys.version_info >= (3, 7):
+        def readline(self, __size: int = ...) -> bytes: ...
+    else:
+        def readline(self, __size: Optional[int] = ...) -> bytes: ...
```

Comment:
Quick testing with Python 3.7 and 3.8 (Linux) shows that readline() also accepts `None`.